doc/sphinx-guides/source/container/dev-usage.rst

```diff
@@ -49,3 +132,56 @@ You can also access the Payara Admin Console if needed, which is available at ht
 
 Note that data is persisted in ``./docker-dev-volumes`` in the root of the Git repo. For a clean start, you should
 remove this directory before running the ``mvn`` commands above.
+
+
+.. _dev-logs:
+
+Viewing Logs
```

Comment:
We could add a snipped/link on how to configure dataverse log levels within docker.

Reply:
Feel free to make a suggestion and/or create an issue that we need docs about that.
It might involve more changes than anticipated, as it will require changes to `logging.properties`, which should not be edited on the overlayfs.